app/code/Magento/Widget/Test/Mftf/Section/AdminNewWidgetSection.xml

```diff
@@ -40,5 +42,10 @@
         <element name="displayMode" type="select" selector="select[id*='display_mode']"/>
         <element name="restrictTypes" type="select" selector="select[id*='types']"/>
         <element name="saveAndContinue" type="button" selector="#save_and_edit_button" timeout="30"/>
+        <!-- Catalog Product List Widget Options -->
```

Comment:
i think it would be best (perhaps) to have a separate section called something like AdminWidgetOptionsSection and it contains all the elements that will appear in that form. the problem is that a lot of those elements already exist on this AdminNewWidgetSection file & i can't move them (cause that would be a major BIC). 

i could create the new section file, add all of my new elements, add all pre-existing elements, deprecate the elements on here that get moved, & update all actiongroups & tests to reference the new section

let me know what you think i should do

Reply:
this is ok as is